Goder.BL/Services/UserService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Goder.DAL.Context;
+using Goder.DAL.Models;
+using AutoMapper;
+using System.Linq;
+
+namespace Goder.BL.Services
+{
+    public class UserService : Abstract.BaseService
+    {
+        public UserService(GoderContext context, IMapper mapper) : base(context, mapper)
+        {
+
+        }
+
+        public User GetUser(Guid id)
+        {
+            return _context.Users.FirstOrDefault(c => c.Id == id);

```

Comment:
Try to use `FirstOrDefaultAsync`. Your method will be `async` and you should use `await` with this method. and you should return `Task<UserDTO>` Your method in controller also should be async and return `Task<ContextResult<UserDTO>>`